microsite/data/plugins/kafka.yaml

```
@@ -0,0 +1,11 @@
+---
+title: Kafka
+author: '@nirga'
+authorUrl: https://github.com/nirga
+category: Monitoring
+description: Observability for Kafka clusters and async API of components.
```

```suggestion
description: Observability for Apache Kafka clusters and async API of components.
```

I've only come across Kafka in some rough industry reading, as I've not worked with a lot of teams using Kafka yet, but I just wondered if it might be valuable to add a reference that it's "Apache Kafka", specifically?  Feel free to suggest this is not necessary. 😁 

I ask because I knew AWS has an offering for it, and in fact interestingly they call it "Amazon Managed Services for **Apache** Kafka", and I just looked and IBM also calls theirs "IBM **Apache** Kafka as a Service", so the big vendors seem to be clearly indicating it's **Apache** Kafka... ? Not sure if that's to clarify vs. other Kafka's, or if Apache Foundation has a copyright on it or something, who knows! So even simply in the description, may help line it up as well as introduce the concept to those maybe not as familiar?

Not sure if that makes sense.

---

Definitely I agree. For me Kafka only means Apache Kafka but it's try, might not be the case for everyone.